linux_os/guide/system/permissions/files/sysctl_fs_protected_symlinks/policy/stig/shared.yml

```diff
@@ -0,0 +1,38 @@
+srg_requirement: |-
+    {{{ full_name }}} must enable kernel parameters to enforce discretionary access control on symlinks.
+
+vuldiscussion: |-
+    By enabling the fs.protected_symlinks kernel parameter, symbolic links are permitted to be followed only when outside a sticky world-writable directory, or when the UID of the link and follower match, or when the directory owner matches the symlink's owner. Disallowing such symlinks helps mitigate vulnerabilities based on insecure file system accessed by privileged programs, avoiding an exploitation vector exploiting unsafe use of open() or creat().
+
+checktext: |-
+    Verify {{{ full_name }}} is configured to enable DAC on symlinks.
+
+    Check the status of the fs.protected_hardlinks kernel parameter with the following command:
+
+    $ sudo sysctl fs.protected_syminks
+
+    fs.protected_symlinks = 1
+
+    If "fs.protected_symlinks " is not set to "1" or is missing, this is a finding.
+
+    Check that the configuration files are present to enable this kernel parameter.
+
+
+    Check that the configuration files are present to enable this kernel parameter.
+
+    $ { /usr/lib/systemd/systemd-sysctl --cat-config; cat /etc/sysctl.conf; } | egrep -v '^(#|$)' | grep -F fs.protected_symlinks | tail -1
+
+    fs.protected_symlinks = 1
+
+    If "fs.protected_symlinks" is not set to "1" or is missing, this is a finding.
+
+fixtext: |-
+    Verify {{{ full_name }}} is configured to enable DAC on symlinks with the following commands:
+
+    Add or edit the following line in a system configuration file in the "/etc/sysctl.d/" directory:
+
+    fs.protected_symlinks = 1
+
+    Load settings from all system configuration files with the following command:
```

Comment:
Why is the `The system configuration files need to be reloaded for the changes to take effect.` removed from hrer?

Reply:
It appears to be a style change from the external reviewer.